st2actions/st2actions/runners/mistral/utils.py

```diff
@@ -101,14 +101,42 @@ def _eval_inline_params(spec, action_key, input_key):
             _merge_dicts(spec[input_key], inputs)
 
 
+def _validate_action_params(spec, action_key, input_key, action):
+
+    if not action or action_key not in spec or spec.get(action_key) != 'st2.action':
+        return
+
+    action_input = spec.get(input_key)
+    action_params = set(action_input.get('parameters', {}).keys())
```

Comment:
If in the line `.get` above returns `None`, this line will throw since you don't pass a default value to get, right?